src/neo-vm/ExecutionEngine.cs

```diff
@@ -289,25 +305,49 @@ private bool ExecuteInstruction()
                     }
                 case OpCode.THROW:
                     {
-                        return false;
+                        if (!TryPop(out StackItem error)) return false;
+                        UncaughtException = error;
```

Comment:
We will allow `Try{ try{`?

Reply:
Yes

Reply:
But if we replace the `UncaughtException`, we will have only the last one, it shouldn't be a stack?

Reply:
How do you throw 2 exceptions at the same time?

Reply:
You are right!